_docs/concepts/traffic-management/rules-configuration.md

```diff
@@ -53,58 +62,91 @@ following important aspects must be kept in mind while writing route rules:
 
 Every rule corresponds to some destination service identified by a
 *destination* field in the rule. For example, all rules that apply to calls
-to the "reviews" service will include the following field.
+to the "reviews" service will include at least the following.
 
 ```yaml
-destination: reviews.default.svc.cluster.local
+destination:
+  name: reviews
 ```
 
-The *destination* value SHOULD be a fully qualified domain name (FQDN). It
-is used by Pilot for matching rules to services. For example,
-in Kubernetes, a fully qualified domain name for a service can be
-constructed using the following format: *serviceName.namespace.dnsSuffix*.
+The *destination* value specifies, implicitly or explicitly, a fully qualified
+domain name (FQDN). It is used by Pilot for matching rules to services.
+
+The FQDN of the service is composed from three components: *name*,
+*namespace*, and *domain*:
+
+```
+FQDN = name + "." + namespace + "." + domain
+```
+
+These fields can be explicitly specified as follows.
+
+```yaml
+destination:
+  name: reviews
+  namespace: default
+  domain: svc.cluster.local
+```
+
+More commonly, to simplify and maximize reuse of the rule (for example, to use
+the same rule in more than one namespace or domain), the rule destination
+specifies only the *name* field, relying on defaults for the other
+two.
+
+The default value for the *namespace* is the namespace of the rule
+itself, which can be specified in the *metadata* field of the rule, or
+at config time using the `-n` option of the CLI. The default value of
```

Comment:
or during rule install using the `istioctl -n <namespace> create -f` or `kubectl -n <namesapce> create -f` command.